tests/test_features.py

```diff
@@ -92,60 +92,61 @@ def test_stone_features(self):
             X.X......
             .........''' + EMPTY_ROW * 7))
 
+        all_zeros = np.zeros([go.N, go.N])
         for i in range(10, 16):
-            self.assertEqualNPArray(f[:, :, i], np.zeros([go.N, go.N]))
+            self.assertEqualNPArray(all_zeros, f[:, :, i])
 
     def test_stone_color_feature(self):
         f = features.stone_color_feature(TEST_POSITION)
-        self.assertEqual(f.shape, (9, 9, 3))
+        self.assertEqual((9, 9, 3), f.shape)
         # plane 0 is B
-        self.assertEqual(f[0, 1, 0], 1)
-        self.assertEqual(f[0, 1, 1], 0)
+        self.assertEqual(1, f[0, 1, 0])
+        self.assertEqual(0, f[0, 1, 1])
         # plane 1 is W
-        self.assertEqual(f[0, 8, 1], 1)
-        self.assertEqual(f[0, 8, 0], 0)
+        self.assertEqual(0, f[0, 8, 0])
+        self.assertEqual(1, f[0, 8, 1])
         # plane 2 is empty
-        self.assertEqual(f[0, 5, 2], 1)
-        self.assertEqual(f[0, 5, 1], 0)
+        self.assertEqual(1, f[0, 5, 2])                                 # < WHAAAAAAATTTT
```

Comment:
it's the "empty" plane - i.e. 1-hot if that spot is neither black nor white. AGZ got rid of the empty planes since in theory it's just 1 - B - W.

Reply:
That makes sense.
I failed to differentiate "stone_color_features" and "stone_features" and thought this part of NEW_FEATURES.